src/ext/oblivc/obliv_bits.c

```diff
@@ -56,7 +56,6 @@ static struct stdioTransport stdioTransport
 void protocolUseStdio(ProtocolDesc* pd)
   { pd->trans = &stdioTransport.cb; }
 
-//#define PROFILE_NETWORK
```

Comment:
Why is it important to have network profiling always enabled? I am okay with it being a runtime flag, which is better than the current state IMO (currently libobliv needs to be rebuilt to enable network profiling). I am also okay with the way libc does it: generate two versions of libobliv, with and without profiling, so that oblivcc can choose which to use based on runtime parameters.